server/services/mappers/alertMapper.ts

```diff
@@ -76,5 +76,6 @@ export const toAlertSummaryData = (alerts: Result<PagedList<Alert>>): AlertSumma
     ...toAlertTypesFilters(alertsContent),
     alertFlags: dpsShared.alertFlags.getAlertFlagLabelsForAlerts(alertsContent),
     apiUnavailable: false,
+    highPublicInterestPrisoner: alertsContent.filter(a => a.alertCode.code === 'HPI' && a.isActive).length > 0,
```

Comment:
Better to use `some` to stop processing when that alert is found and just make the code a bit clearer
```suggestion
    highPublicInterestPrisoner: alertsContent.some(a => a.alertCode.code === 'HPI' && a.isActive),
```